app/styles/_core.less

```diff
@@ -978,6 +978,16 @@ a.disabled-link {
     margin-top: 0px;
     margin-bottom: 10px;
   }
+  .indent {
```

Comment:
These styles are specific to the home page dialog and shouldn't be in core.less or have such generic names.